src/components/text-note/footer/EmojiPickerPanel.tsx

```
@@ -1,23 +1,37 @@
   onEmojiSelect: (emoji: string) => void
 }
 
 export default function EmojiPickerPanel({ onEmojiSelect }: Props) {
+  const emojiPickerRef = useRef<Picker>(null)
+
+  useEffect(() => {
+    const element = emojiPickerRef.current
+    if (!element) return
+
+    const handleEmojiClick = (event: EmojiClickEvent) => {
+      onEmojiSelect(event.detail.unicode ?? '')
```

ESLint

[error] Unsafe argument of type error typed assigned to a parameter of type `string`.

(ts/no-unsafe-argument)